src/app/client/src/app/modules/shared-feature/components/onboarding-user-selection/onboarding-user-selection.component.ts

```diff
@@ -48,10 +50,22 @@ export class OnboardingUserSelectionComponent implements OnInit, OnDestroy {
     private formService: FormService,
     private profileService: ProfileService,
     private userService: UserService,
-    private toasterService: ToasterService
+    private toasterService: ToasterService,
+    private popupControlService: PopupControlService
   ) { }
 
   ngOnInit() {
+    /**
```

Comment:
@mithun30052001 refactor this comment Here we enable/disable the usertype popup based on the isvisible value returned from the form config request 
      *                along with setting up the required localstorage values in the case usertype popup is disabled
    */

Reply:
comment refactored @princegupta1131